src/molecules/Product.tsx

```diff
@@ -36,15 +36,15 @@ interface ProductProps {
   image: string
   secondImage: string
   price: number
-  productName: string
+  product: string
```

Comment:
Maybe let's use `name` instead? So then it would be product's name (`Product.name`) when you use it. Right now it's `Product.product`, which may be confusing.